application/src/main/kotlin/dev/hypest/pis/infrastructure/db/DbOrderQuery.kt

```diff
@@ -43,11 +47,25 @@ class DbOrderQuery(
                     address = it.deliveryDetails.address
                 )
             },
-            isFinalized = draftOrder.isFinalized,
-            isPaid = orderToPay?.isPaid ?: false,
-            isConfirmed = activeOrder?.isConfirmed ?: false,
-            isReadyToDeliver = activeOrder?.isReadyToDeliver ?: false,
-            assignedCourierId = orderDelivery?.assignedCourierId
+            status = getStatus(draftOrder, orderToPay, activeOrder, orderDelivery)
         )
     }
+
+    private fun getStatus(
+        draftOrder: DraftOrderEntity,
+        orderToPay: OrderToPayEntity?,
+        activeOrder: ActiveOrderEntity?,
+        orderDelivery: OrderDeliveryEntity?
+    ) = when {
+//        orderDelivery?.isDelivered -> OrderResponse.OrderStatus.DELIVERED
+//        orderDelivery?.isBeingDelivered -> OrderResponse.OrderStatus.DELIVERY_IN_PROGRESS
```

Comment:
to chyba mozna odkomentowac

Reply:
Tylko Twoje najpierw zmergować i sobie maina zrebasuje

Reply:
a faktoza xD już merguje